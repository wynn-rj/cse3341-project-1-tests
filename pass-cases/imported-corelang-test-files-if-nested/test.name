if nested (Imported from corelan-test-files)
