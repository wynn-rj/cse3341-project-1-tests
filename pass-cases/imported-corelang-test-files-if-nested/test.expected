program
  int b;
begin
  b:=1;
  if b=1 then
    output b;
    if b=1 then
      b:=2;
      output b;
    endif;
    output b;
  endif;
end
1
2
2
